exonum-java-binding/package_app.sh

```diff
@@ -46,8 +46,8 @@ function build-exonum-java-for-platform() {
 function build-exonum-java-macos() {
     # We use static linkage for RocksDB on Mac because in case of dynamic linking
     # the resulting app has a dependency on a _particular_
```

Comment:
@vitvakatu it is not true — haven't you clarified that soname/install_name do not contain the same version string as the library? Please fix misleading statements in the docs.

Reply:
It usually contains the minor version of the library (but not in case of libsodium). Updated, how about current comment?

Reply:
It's good, thanks. I've updated it slightly.